ingestion/functions/parsing/india/india.py

```diff
@@ -32,70 +33,116 @@ def convert_gender(raw_gender):
         return "Male"
     if raw_gender == "F":
         return "Female"
-    return None
 
 
-def convert_location(raw_entry):
-    state = raw_entry["detectedstate"]
-    district = raw_entry["detecteddistrict"]
-    city = raw_entry["detectedcity"]
+def convert_location(row):
+    state = row["Detected State"]
+    district = row["Detected District"]
+    city = row["Detected City"]
 
     query_terms = ("India",)
-    location = {"country": "India"}
+    resolutions = ("Country",)
     if state:
-        location["administrativeAreaLevel1"] = state
         query_terms = (state,) + query_terms
+        resolutions = ("Admin1",) + resolutions
     if district:
-        location["administrativeAreaLevel2"] = district
         query_terms = (district,) + query_terms
+        resolutions = ("Admin2",) + resolutions
     if city:
-        location["administrativeAreaLevel3"] = city
         query_terms = (city,) + query_terms
+        resolutions = ("Admin3",) + resolutions
 
-    location["query"] = ", ".join(query_terms)
-    return location
+    return {
+        "query": ", ".join(query_terms),
+        "limitToResolution": ",".join(resolutions)
```

Comment:
you already add `,` to the resolutions above, it'd simpler if you just added the resolution names in an array and joined them here as you do

Reply:
SG, done.